### PR TITLE
add footer

### PR DESCRIPTION
**Pull request by CoreX20**

### CODE REVIEW

**Review by ardyanandakirana — COMMENTED** [review]
please add a href contact

**Comment by CoreX20** [comment]
done

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          <p>Jalan Suroyo No. 161 Mayangan Kota Probolonggo 672000</p>
+          <p>binarcarrental@gmail.com</p>
+          <p>081233334808</p>
+        </div>
+        <div class="col-2 footer__nav">
+          <div class="d-flex flex-column justify-content-center gap-4 fw-bold">
+            <a href="#our-services">Our Services</a>
+            <a href="#why-us">Why Us</a>
+            <a href="#testimonial">Testimonial</a>
+            <a href="#faq">FAQ</a>
```
Comment (ardyanandakirana) [inline index.html:27]: please, add a href contact

Comment (CoreX20) [reply index.html:27]: done

**Pull request merged**